src/VscPreviewExtension/src/extension.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+"use strict";
+
+import { workspace, window, ExtensionContext, commands, Event, Uri, ViewColumn, TextDocument } from "vscode";
+import * as path from "path";
+import { PreviewCore } from "./PreviewCore";
```

Comment:
>PreviewCore [](start = 31, length = 11)

minor: for typescript, class name "PreviewCore" is correct, but file name should start with lower case like "previewCore.ts"? and consider to use "MarkdownDocumentContentProvider" for better class name?